src/main/java/org/spongepowered/asm/util/Bytecode.java

```diff
@@ -1199,16 +1257,13 @@ public static String getUnboxingMethod(Type type) {
      * @param b Incoming method
      */
     public static void compareBridgeMethods(MethodNode a, MethodNode b) {
-        ListIterator<AbstractInsnNode> ia = a.instructions.iterator();
-        ListIterator<AbstractInsnNode> ib = b.instructions.iterator();
+        ListIterator<AbstractInsnNode> ia = ListIterators.filter(a.instructions.iterator(), Bytecode::isRealInsn);
+        ListIterator<AbstractInsnNode> ib = ListIterators.filter(b.instructions.iterator(), Bytecode::isRealInsn);
```

Comment:
They added this filtering, so the iterators became normal `Iterator` instances. Our changes below are using `.previous()`, which is only in `ListIterator`. I had to make that `ListIterators` class to make it support our changes (I tested that part intensively (random 1B ops) and made sure it was acting exactly like a normal `ListIterator`)